docs/user/build-customization.rst

```diff
@@ -104,6 +105,60 @@ To avoid this, it's possible to unshallow the clone done by Read the Docs:
          - git fetch --unshallow
 
 
+Cancel build based on a condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When a command exits with code ``183``,
```

Comment:
We should probably explain why we chose 183, since it definitely seems somewhat arbitrary :)

Reply:
I added a small note for this.